### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by cdlindsey**
Slides from SciPy 2023

If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.

*Editor*:  

*Reviewers*:

### CODE REVIEW

File: presentations/slides/info.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "title": "bayes_mapvar: Bayesian Statistics with Python, No Resampling Necessary",
+    "authors": [
+        {
+            "name": "Charles Lindsey",
+            "affiliation": "Revionics, an Aptos Company",
+        },
```
Comment (deniederhut) [inline presentations/slides/info.json:8]: Looks like we're missing the orcid field here? https://github.com/scipy-conference/scipy_proceedings#slideposter-submission-steps

**Comment by deniederhut** [comment]
@scoobies mark not ready

File: presentations/slides/info.json (hunk 1 above)
Comment (cdlindsey) [reply presentations/slides/info.json:8]: just added. I think.

Comment (deniederhut) [reply presentations/slides/info.json:8]: Thanks!

**Review by deniederhut — APPROVED** [review]
giphy-4271990831

**Pull request merged**